bound/typescript/src/wasm/mappings.ts

```diff
@@ -0,0 +1,787 @@
+import wasm from "./";
```

Comment:
I kind of forgot, but this is generated right? Probably should rename to `generated-mappings.ts` so no one tries to change this manually